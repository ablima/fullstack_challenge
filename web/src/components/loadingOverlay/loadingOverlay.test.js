import React from 'react';
import renderer from 'react-test-renderer';
import LoadingOverlay from './index';

test('Check if render is correct', () => {
  const component = renderer.create(
    <LoadingOverlay />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});