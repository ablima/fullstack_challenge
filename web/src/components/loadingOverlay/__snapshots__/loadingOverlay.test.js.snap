// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if render is correct 1`] = `null`;
